### PR TITLE
Issue 151 - fixing bugs

### PR DESCRIPTION
**Pull request by hani-ibrahim**
follow up for issue #151 
- fixing tapping the payment selection after the order is placed
- trimming the address 
- Adding needed framework to run the app on the device

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: AtlasUI/AtlasUI/CheckoutSummaryStoryboardViewController.swift [hunk 1]
```diff
@@ -91,13 +91,17 @@ class CheckoutSummaryStoryboardViewController: UIViewController, CheckoutProvide
 
     @IBAction private func shippingAddressTapped() {
         // TODO when addresses are implemented
+        guard viewState.showDetailArrow else { return }
     }
 
     @IBAction private func billingAddressTapped() {
         // TODO when addresses are implemented
+        guard viewState.showDetailArrow else { return }
```
Comment (bonkey) [inline AtlasUI/AtlasUI/CheckoutSummaryStoryboardViewController.swift]: Seems like these two methods do nothing. Maybe in `UserMessage`

```
func notImplemented() {
  AtlasLogger.logError(...)
  show(...)
}
```

and call `userMessage.notImplemented()` here or something like this?

File: AtlasUI/AtlasUI/Utils/String.swift [hunk 2]
```diff
@@ -0,0 +1,13 @@
+//
+//  Copyright © 2016 Zalando SE. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+
+    var trimmedString: String {
```
Comment (bonkey) [inline AtlasUI/AtlasUI/Utils/String.swift]: Type in name is redundant, just `trimmed`

Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/Utils/String.swift]: I guess It's a code style for swift
checking String properties, there are `lowercaseString` and `uppercaseString` 
what do you think?

Comment (bonkey) [inline AtlasUI/AtlasUI/Utils/String.swift]: For ancient Swift, prior to magnificent version 3 :-)

https://github.com/apple/swift/blob/swift-3.0-branch/stdlib/public/core/String.swift#L1040

It doesn't conflict, reads better and it's future. Let's go modern.

Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/Utils/String.swift]: Good catch 👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

**Pull request merged**